documentation/index.md

```diff
@@ -228,14 +228,14 @@ def replicate_list(val):
 
 @dag
 def pipeline():
-  v1, v2, v3, v4 = replicate_tuple(1)
-  v5, v6, v7, v8 = replicate_list(v4)
+  v1, v2, v3, v4 = replicate_tuple(1)  # notice unpacked values here
```

Comment:
I'm not sure these comments really add anything to it

Reply:
better this way